### DISCUSSION TITLE
Ignore environment certificates?

### DISCUSSION THREAD

**Discussion by tpnoonan** [Dynatrace/dynatrace-configuration-as-code#766] [Q&A] [answered]
I've got a situation where my API client is only allowed to communicate with a managed node or CAG using IP address to connect.  For my own purposes, I just have curl ignore certificates.  Is there a switch/setting for monaco available to have it ignore certificates for the environments that it is connecting to?

**Comment by UnseenWizzard** [comment, marked as answer]
Hi @tpnoonan. 
This is not possible inside monaco and not planned to be added as a feature. 

However you can solve this the following way: 

> monaco is using the host system trust store to determine if SSL is trusted. 
> If you add self-signed certificate to the hosts trust store, monaco will accept it.

see also https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/discussions/291
